gobblin-yarn/src/main/java/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -545,6 +545,8 @@ public static void main(String[] args) throws Exception {
       Log4jConfigurationHelper.updateLog4jConfiguration(
           GobblinApplicationMaster.class, Log4jConfigurationHelper.LOG4J_CONFIGURATION_FILE_NAME);
 
+      LOGGER.info(YarnHelixUtils.getJvmInputArguments());
```

Comment:
I see. So the client app will need to parse the logs (can you also include this somewhere?). Could we abstract this into a broadcast method which by default just logs, but can be overriden for other broadcast implementations (I could see written to a pid-like file, or using Zookeeper).